hermes-common/src/main/java/pl/allegro/tech/hermes/common/metric/WorkloadMetrics.java

```diff
@@ -0,0 +1,176 @@
+        gaugeRegistrar.registerGauge(
+                "consumers-workload." + kafkaCluster + ".missing-resources",
+                "workload.missing-resources",
+                obj,
+                f,
+                Tags.of(KAFKA_CLUSTER_TAG, kafkaCluster)
+        );
+    }
+
+    public <T> void registerDeletedAssignmentsGauge(T obj, String kafkaCluster, ToDoubleFunction<T> f) {
+        gaugeRegistrar.registerGauge(
+                "consumers-workload." + kafkaCluster + ".deleted-assignments",
+                "workload.deleted-assignments",
+                obj,
+                f,
+                Tags.of(KAFKA_CLUSTER_TAG, kafkaCluster)
+        );
+    }
+
+    public <T> void registerCreatedAssignmentsGauge(T obj, String kafkaCluster, ToDoubleFunction<T> f) {
+        gaugeRegistrar.registerGauge(
+                "consumers-workload." + kafkaCluster + ".created-assignments",
+                "workload.created-assignments",
+                obj,
+                f,
+                Tags.of(KAFKA_CLUSTER_TAG, kafkaCluster)
+        );
+    }
+
+    public HermesTimer rebalanceDurationTimer(String kafkaCluster) {
+        return HermesTimer.from(
+                meterRegistry.timer("workload.rebalance-duration", Tags.of(KAFKA_CLUSTER_TAG, kafkaCluster)),
+                hermesMetrics.consumersWorkloadRebalanceDurationTimer(kafkaCluster)
+        );
+    }
+
+    public <T> void registerRunningSubscriptionsGauge(T obj, ToDoubleFunction<T> f) {
+        gaugeRegistrar.registerGauge("consumers-workload.monitor.running", "workload.subscriptions.running", obj, f);
+    }
+
+    public <T> void registerAssignedSubscriptionsGauge(T obj, ToDoubleFunction<T> f) {
+        gaugeRegistrar.registerGauge("consumers-workload.monitor.assigned", "workload.subscriptions.assigned", obj, f);
+    }
+
+    public <T> void registerMissingSubscriptionsGauge(T obj, ToDoubleFunction<T> f) {
+        gaugeRegistrar.registerGauge("consumers-workload.monitor.missing", "workload.subscriptions.missing", obj, f);
+    }
+
+    public <T> void registerOversubscribedGauge(T obj, ToDoubleFunction<T> f) {
+        gaugeRegistrar.registerGauge("consumers-workload.monitor.oversubscribed", "workload.subscriptions.oversubscribed", obj, f);
+    }
+
+    public <T> void registerOperationsPerSecondGauge(T obj, ToDoubleFunction<T> f) {
+        gaugeRegistrar.registerGauge("consumer-workload.weighted.load.ops", "workload.weighted.ops", obj, f);
+    }
+
+    public <T> void registerCpuUtilizationGauge(T obj, ToDoubleFunction<T> f) {
+        gaugeRegistrar.registerGauge("consumer-workload.weighted.load.cpu-utilization", "workload.weighted.cpu-utilization", obj, f);
+    }
+
+    public <T> void registerCurrentScoreGauge(String consumerId, T obj, ToDoubleFunction<T> f) {
+        gaugeRegistrar.registerGauge(
+                buildFullGraphiteMetricPath(CURRENT_SCORE, consumerId),
+                "workload.weighted.current-error",
```

Comment:
suggestion: If we are planning to use the name `current-error` instead of `current-score` in the future, maybe we should already rename the method name to reflect that?